### PR TITLE
Feat/provider registration

### PR DESCRIPTION
**Pull request by Harr-izle**

### CODE REVIEW

File: src/app/features/pro-registration/components/profile-update/profile-update.component.ts [hunk 1]
```diff
@@ -22,4 +28,19 @@ export class ProfileUpdateComponent {
     })
     dialogRef.afterClosed().subscribe((results) => console.log(results))
   }
+  triggerFileInput(): void {
+    document.getElementById('file')?.click();
+  }
+
+  onFileSelected(event: any): void {
+    const file = event.target.files[0];
```
Comment (ernestbrayce) [inline src/app/features/pro-registration/components/profile-update/profile-update.component.ts]: Add null safety here

`if (input?.files?.length) {`

File: src/app/features/pro-registration/components/profile-update/profile-update.component.ts [hunk 2]
```diff
@@ -22,4 +28,19 @@ export class ProfileUpdateComponent {
     })
     dialogRef.afterClosed().subscribe((results) => console.log(results))
   }
+  triggerFileInput(): void {
+    document.getElementById('file')?.click();
+  }
+
+  onFileSelected(event: any): void {
```
Comment (ernestbrayce) [inline src/app/features/pro-registration/components/profile-update/profile-update.component.ts]: Stop using type `any`

**Review by ernestbrayce — APPROVED** [review]
Fix the comments and merge

File: src/app/features/pro-registration/components/payment-method/payment-method.component.ts [hunk 3]
```diff
@@ -10,22 +12,37 @@ interface paymentMethod {
...
+  templateUrl:'./payment-method.component.html',
   styleUrl: './payment-method.component.sass'
 })
 export class PaymentMethodComponent {
-  paymentMethod!: paymentMethod[];
+  paymentMethod: paymentMethod [] =[
+    {name: 'Mobile Money'},
+    {name: 'Bank Account'},
+    {name: 'Paypal Account'},
+  ]
```
Comment (BlissArmah-Nwanwah) [inline src/app/features/pro-registration/components/payment-method/payment-method.component.ts:24]:
```suggestion
  paymentMethod =[
    {name: 'Mobile Money'},
    {name: 'Bank Account'},
    {name: 'Paypal Account'},
  ]
```

**Review by BlissArmah-Nwanwah — APPROVED** [review]

File: src/app/features/pro-registration/components/payment-method/payment-method.component.ts (hunk 3 above)
Comment (BlissArmah-Nwanwah) [inline src/app/features/pro-registration/components/payment-method/payment-method.component.ts]: Typescript will automatically infer the type from the value assigned to the variable (Payment Method) so there is no need to explicitly type it 

```suggestion
  paymentMethod =[
    {name: 'Mobile Money'},
    {name: 'Bank Account'},
    {name: 'Paypal Account'},
  ]
```


Also you can create a constant for your values and use them here

**Pull request merged**